redvanworkshop/website
# RBD-7 Swap the Logo to new Horizontal RVW

#### What's this PR do?

- Removed old text logos and a couple pieces of associated styling
- Created a new svg horizontal logo svg's in a couple variations of white and black text
- Adjusted navbar height to incorporate the logo a little more comfortably

#### Where should the reviewer start?

Once deployed to development, the navbar changes can be reviewed on any page.

#### How should this be manually tested?

Please help me double check if there's anything cross-browser or responsive I may have missed!  The navbar changes as you scroll down as well as when you switch browser widths responsively. 

#### Any background context you want to provide?

There wasn't a lot of guidance in this ticket.  Peter and I played with quite a few variations before arriving that this is probably the best path forward in a particularly difficult request.  Please feel free to provide feedback!

#### What are the relevant github issue?

No

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/40673946/72105761-290bad80-32fc-11ea-971c-d6f6c2787012.png)
![image](https://user-images.githubusercontent.com/40673946/72105797-39238d00-32fc-11ea-82e8-ff8dbfbf97cb.png)

#### What gif best describes this PR or how it makes you feel?

![image](https://user-images.githubusercontent.com/40673946/72105913-6a9c5880-32fc-11ea-8739-67ade6b4e2e7.png)

#### Definition of Done:

- [ ] You have verified that either `npm test` or `npm run build` passes without issue
![image](https://user-images.githubusercontent.com/40673946/72106569-9cfa8580-32fd-11ea-8864-d5b02a14cdcd.png) ** it doesn't... but not on any of the files I've changed.  I believe this file just needs to be added to an ignore list.
